modules/install-ibm-cloud-preparing-the-provisioner-node.adoc

```diff
@@ -0,0 +1,219 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_cloud/install-ibm-cloud-installing-on-ibm-cloud.adoc
+
+[id="preparing-the-provisioner-node-for-openshift-install-on-ibm-cloud_{context}"]
+= Preparing the provisioner node for {product-title} installation on IBM Cloud
+
+Perform the following steps to prepare the environment.
+
+.Procedure
+
+. Log in to the provisioner node via `ssh`.
+
+. Create a non-root user (`kni`) and provide that user with `sudo` privileges:
++
+[source,terminal]
+----
+# useradd kni
+----
++
+[source,terminal]
+----
+# passwd kni
+----
++
+[source,terminal]
+----
+# echo "kni ALL=(root) NOPASSWD:ALL" | tee -a /etc/sudoers.d/kni
+----
++
+[source,terminal]
+----
+# chmod 0440 /etc/sudoers.d/kni
+----
+
+. Create an `ssh` key for the new user:
++
+[source,terminal]
+----
+# su - kni -c "ssh-keygen -t ed25519 -f /home/kni/.ssh/id_rsa -N ''"
+----
+
+. Log in as the new user on the provisioner node:
++
+[source,terminal]
+----
+# su - kni
+$
+----
+
+. Use Red Hat Subscription Manager to register the provisioner node:
++
+[source,terminal]
+----
+$ sudo subscription-manager register --username=<user> --password=<pass> --auto-attach
+----
++
+[source,terminal]
+----
+$ sudo subscription-manager repos --enable=rhel-8-for-x86_64-appstream-rpms --enable=rhel-8-for-x86_64-baseos-rpms
+----
++
+[NOTE]
+====
+For more information about Red Hat Subscription Manager, see link:https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html-single/rhsm/index[Using and Configuring Red Hat Subscription Manager].
+====
+
+. Install the following packages:
++
+[source,terminal]
+----
+$ sudo dnf install -y libvirt qemu-kvm mkisofs python3-devel jq ipmitool
+----
+
+. Modify the user to add the `libvirt` group to the newly created user:
++
+[source,terminal]
+----
+$ sudo usermod --append --groups libvirt <user>
```

Comment:
This may as well use "kni" as we have hardcoded kni in other places, no need to switch to <user> all of a sudden 
$ sudo usermod --append --groups libvirt kni